lib/sqlalchemy/testing/suite/test_types.py

```diff
@@ -452,6 +453,123 @@ def test_concatenate_clauselist(self, connection):
         )
 
 
+class IntervalTest(_LiteralRoundTripFixture, fixtures.TablesTest):
+    __requires__ = ("datetime_interval",)
+    __backend__ = True
+    compare = None
+    datatype = Interval
+    data = datetime.timedelta(days=1, seconds=4)
+
+    @classmethod
+    def define_tables(cls, metadata):
+        class Decorated(TypeDecorator):
+            impl = cls.datatype
+            cache_ok = True
+
+        Table(
+            "interval_table",
+            metadata,
+            Column(
+                "id", Integer, primary_key=True, test_needs_autoincrement=True
+            ),
+            Column("interval_data", cls.datatype),
+            Column("date_data", DateTime),
+            Column("decorated_interval_data", Decorated),
+        )
+
+    @testing.combinations(
+        (
```

Comment:
Hi, 
this is minor but just wanted to bring it up here, 
this testcase was failing with for days > 99days {Precision of more than 2 digits } with oracle DB for round trip test, due #10509, ig we don't have this testcase now with that scenario, in PR for #10513 only compilation tests are added not the round trip as literal processor thing which was being added in this PR.

Reply:
Yeah sure, 

So at that time Interval Parameter were ineffective for oracleDB due to issue-10509, 
But my Interval param were `day_precision=9` and `second_precision=9`, 

Now this will work since PR-10513 is merged.

the testcase was this

```py
    @testing.combinations(
        (
            datetime.timedelta(days=103, seconds=4),
            datetime.timedelta(days=103, seconds=4),
        ),
        argnames="input_, compare_",
    )
    def test_render_literal_interval(
        self, literal_round_trip, input_, compare_
    ):
        literal_round_trip(Interval(), [input_], [compare_])
```

i remvoed the 3 digit day test case to make CI work for simple testcases in this PR.

Reply:
OK that is restored in  https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4939 if you want to review

Reply:
Yep looks good!